lua/nvim-treesitter/parsers.lua

```diff
@@ -926,10 +926,10 @@ list.markdown_inline = {
 
 list.matlab = {
   install_info = {
-    url = "https://github.com/mstanciu552/tree-sitter-matlab",
-    files = { "src/parser.c" },
+    url = "https://github.com/acristoffers/tree-sitter-matlab",
```

Comment:
```suggestion
    url = "https://github.com/acristoffers/tree-sitter-matlab",
    branch = "main",
```

Reply:
Shouldn't be required if `main` is the default branch, but the download currently fails. (May just be a hiccup on GH's side, though; let's give the archive a bit of time to propagate through the CDN. Making changes in the repo and here at the same time will interfere with CI.)